### PR TITLE
[Feat]:Text Utilities added

### PR DESCRIPTION
**Pull request by t007rushi**
Text Utilities added for typography:

Heading from H6 to H1
Alignment for particular text
text-decoration used
Different sizes of text from Small to large

### CODE REVIEW

File: components/txt-util.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.txt-lg {
+  font-size: 2rem;
+}
+.txt-md {
+  font-size: 1.5rem;
+}
+.txt-sm {
+  font-size: 0.5rem;
```
Comment (bossmonkey98) [inline components/txt-util.css:38]: use px if its less than 10px

File: components/txt-util.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+.txt-underline {
+  text-decoration: underline;
+}
+
+.txt-lg {
+  font-size: 2rem;
+}
+.txt-md {
+  font-size: 1.5rem;
```
Comment (bossmonkey98) [inline components/txt-util.css:35]: And extra spacing is not required

**Review by bossmonkey98 — COMMENTED** [review]
Baki sab Okay

**Review by horsemaker — APPROVED** [review]
looks good! 👌

**Review by bossmonkey98 — COMMENTED** [review]
Aproved

**Review by bossmonkey98 — APPROVED** [review]

**Pull request merged**